### PR TITLE
[Translation] [Portuguese Brazil]: algorithms-data

### PR DESCRIPTION
**Pull request by rschio**
Translation of file algorithms-data.article to brazilian portuguese.

### CODE REVIEW

File: _content/tour/por/algorithms-data.article [hunk 1]
```diff
@@ -42,16 +42,16 @@ This sample program implements a basic hash table.
...
-- See more at [[https://en.wikipedia.org/wiki/Linked_list][https://en.wikipedia.org/wiki/Linked_list]]
+- Veja mais em [[https://en.wikipedia.org/wiki/Linked_list][https://en.wikipedia.org/wiki/Linked_list]]
 
-*Diagram*
+*Diagrama*
 
-    With a linked list, values are tied together in different
-    order through the use of pointers.
+    Com uma lista encadeada, os valores são vinculados entre si
+    em ordem diferente, por meio do uso de ponteiros.
```
Comment (rschio) [inline _content/tour/por/algorithms-data.article:54]: I didn't understand this part: "in different order", maybe someone can help.

Comment (luciormoraes) [reply _content/tour/por/algorithms-data.article:54]: The values are not stored in contiguous block of memory, like an array. Linked lists store values at different memory locations, so they are linked together througnt these pointers.

Comment (rschio) [reply _content/tour/por/algorithms-data.article:54]: Do you think "Com uma lista encadeada, os valores ficam em diferentes locais da memória e são vinculados por meio do uso de ponteiros." would fit better?

Comment (luciormoraes) [reply _content/tour/por/algorithms-data.article:54]: Nice one @rschio

**Review by luciormoraes — APPROVED** [review]

**Pull request merged**